src/server/components/formRunner/funeralDirectors.ts

```diff
@@ -15,10 +15,10 @@ const FormRunnerFields: { [key: string]: string } = {
   speakEnglish: "jsonData.speakEnglish",
   contactName: "jsonData.contactName",
   organisationName: "jsonData.organisationName",
-  "address.firstLine": "address.firstLine",
-  "address.secondLine": "address.secondLine",
-  city: "address.city",
-  postCode: "address.postCode",
+  "address.firstLine": "jsonData['address.firstLine']",
```

Comment:
it's implemented like this so that you are always taking the canonical address (essentially validated by AWS Location Services). `ListItem.Address` rather than `ListItem.jsonData.["address.xx"]`. 

I suggest we merge updated address fields with listItem.address, similar to how we merge updatedJsonData to jsonData in `initialiseFormRunnerSession`.

```
  listItem.jsonData = merge(listItem.jsonData, listItemJsonData.updatedJsonData ?? {})

  listItem.address = {
    ...listItem.address,
    ...getChangedAddressFields(listItem.jsonData, listItem.address)
  }  
```

Reply:
sure.  This has been converted to use the canonical address object